lib/jekyll/webmention_io.rb

```diff
@@ -351,13 +351,14 @@ def self.uri_ok?(uri)
 end
 
 # Load all the bits
-Dir[File.expand_path("commands/*.rb", __dir__)].each do |file|
-  require file
-end
-Dir[File.expand_path("generators/*.rb", __dir__)].each do |file|
-  require file
-end
-require "#{__dir__}/tags/_.rb"
-Dir[File.expand_path("tags/*.rb", __dir__)].each do |file|
-  require file unless file.include? "_.rb"
+def require_all(group)
+  Dir[File.expand_path("#{group}/*.rb", __dir__)].each do |file|
+    require file
+  end
 end
+
+require_all "commands"
+require_all "generators"
+
+require_relative "tags/_.rb"
```

Comment:
Will Ruby automatically skip requiring `_.rb` a second time when `require_all` is called and the directory is looped?

Reply:
Yes, the `require*` macros are optimized that way by checking against a global object `$LOADED_FEATURES`

Reply:
Ok cool.